### DISCUSSION TITLE
American States, can they be forced? How do I do it?

### DISCUSSION THREAD

**Discussion by LAwmand** [magicbug/Cloudlog#2105] [General]
Hi Ive started to look at Worked All States on 23cm now I have 8 left to go.

Some of the stations I've worked for new states have been "roving" so the same callsign is worked in multiple states. Cloudlog seems to get states from QRZ or the grid, I'm not sure. (the grid could refer to 4 states). 

There is no box for adding a state in the log overview as far as I can see. In the advanced log it is shown, but no editable.

Is there any way to input states manually in Cloudlog other than exporting all American contacts, editing them externally and then reimporting them?
David

**Comment by phl0** [comment]
Not sure if I am missing something but there is an input field for US State in the "General" tab while logging the QSO. Isn't that what you are looking for?

![Screenshot from 2023-05-02 15-11-34](https://user-images.githubusercontent.com/7112907/235676310-2265af52-11d8-4d15-9dc8-59d539360f68.png)

**Comment by LAwmand** [comment]
Thanks Florian,
I wasn't aware. I import most of my contacts but thanks to you I've found
the tabs on edit QSO and under awards is the state so I can do it
retrospectively.
Admin. This thread can be closed.
David

**Reply by phl0** [reply]
> Admin. This thread can be closed. David

Can't you mark it as answered? :)

**Comment by LAwmand** [comment]
Closed, Answered
